src/cortex_cli/cortex_cli.py

```diff
@@ -378,6 +362,55 @@ def _validate_cortex_cli_auth_login(no_daemon, no_refresh, config_file) -> dict:
     return config
 
 
+def _refresh_tokens(  # pylint: disable=too-many-arguments
+    refresh_period: int,
+    no_daemon: bool,
+    no_refresh: bool,
+    config: ConfigFile,
+    auth_server_url: str,
+    realm: str,
+    client_id: str,
+    tokens_file: str,
+    refresh_token: str,
```

Comment:
If tokens file exists and is valid `refresh_token` can (should?) be read from there. Why is it specified here separately?

Reply:
Changed accordingly